### PR TITLE
Buildcraft 7. Modular texture directories.

### PR DESCRIPTION
**Pull request by Evolution0**
This adds the new BC7 textures to todo minus ones that were the same as BC6 and could be copied over.

This does not touch the existing buildcraft directory (minus a texture or two that were not 32x which I moved to todo), it only adds the new modular directory structure and copies what textures are re-usable to them.

Even if textures are similar they are *not* the same, I double checked. BC7 textures are less noisy, saturated, and have less contrast. The new textures need to be done from scratch.

tl;dr: BC6 support is still there, this just makes way for BC7 support.

### CODE REVIEW

**Comment by StefanTT** [comment]
:+1:

**Comment by Evolution0** [comment]
This is actually holding other commits back as I don't want to stack PRs.
